src/test/java/com/crud/tasks/trello/client/TrelloMapperTestSuite.java

```diff
@@ -0,0 +1,126 @@
+package com.crud.tasks.trello.client;
+
+import com.crud.tasks.domain.*;
+import com.crud.tasks.mapper.TrelloMapper;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Arrays;
+import java.util.List;
+
+@RunWith(MockitoJUnitRunner.class)
+public class TrelloMapperTestSuite {
+
+    @InjectMocks
```

Comment:
trelloMapper does not have any dependencies, so why do you usd InjectMocks annotation?